src/overview/graph-links.ts

```diff
@@ -0,0 +1,78 @@
+import { DependencyLink, NodeSelection } from '../components/types';
```

Comment:
What is the meaning of overview? What functionalities are stored here?

Reply:
I named the main screen (the one with force graph) "overview". I have grouped files per view (overview, details)